java/integTest/src/test/java/glide/modules/VectorSearchTests.java

```diff
@@ -330,4 +330,49 @@ public void ft_drop() {
         assertInstanceOf(RequestException.class, exception.getCause());
         assertTrue(exception.getMessage().contains("Index does not exist"));
     }
+
+    @SneakyThrows
+    @Test
+    public void ft_aliasadd_aliasdel_aliasupdate() {
+
+        var alias1 = "a1";
+        var alias2 = "a2";
+        var indexName = "index";
```

Comment:
The index name and alias name are not unique enough.

Reply:
good catch